fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -4121,9 +4125,13 @@ struct StartFullRestoreTaskFunc : RestoreTaskFuncBase {
 		// Order does not matter, they will be put in order when written to the restoreFileMap below.
 		state std::vector<RestoreConfig::RestoreFile> files;
 
+		Version firstConsistentVersion = std::numeric_limits<int64_t>::min();
```

Comment:
The default here should be `beginVersion`.  There is a restore mode which ignores range files and only applies logs, in which case the RangeFile list will be empty, which would result in a large negative number, but from the restore's perspective the applied changes are consistent starting with `beginVersion` if there are no range files.

Reply:
Thank you, done.